assets/layout/common/_contact_panels.scss

```diff
@@ -4,18 +4,10 @@
   height: auto;
   margin-top: 40px;
   border-top: $default-border-thin;
-
+  display: flex;
+  flex-direction: column;
   @include respond-to($mq-l) {
-    &:after {
-      content: "";
-      position: absolute;
-      top: 0;
-      bottom: $baseline-unit*4;
-      left: 50%;
-      border-right: 1px solid $color-newsletter-input-border;
-      width: 1px;
-      min-height: 400px;
```

Comment:
I think this - or something like it - should be kept: I'm guessing we just need to set the value for `left` to 33.3% to restore the vertical rules between columns.

Reply:
The borders have been adjusted now